smoke-test/go.sum

```diff
@@ -1,9 +1,10 @@
 github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
-github.com/google/uuid v1.2.0 h1:qJYtXnJRWmpe7m/3XlyhrsLrEURqHRM2kxzoxXqyUDs=
 github.com/google/uuid v1.2.0/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
-github.com/mailru/go-clickhouse v1.8.0 h1:wqTHVsfR4g+BSwKso7X90RdOsVXaSwqJ96GmgBPSgVA=
-github.com/mailru/go-clickhouse v1.8.0/go.mod h1:crHi+yrqslIClnYPm8IOxYVX6GmYVYymJ601I4jDqvo=
+github.com/google/uuid v1.6.0 h1:NIvaJDMOsjHA8n1jAhLSgzrAzy1Hgr+hNrb57e+94F0=
+github.com/google/uuid v1.6.0/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
+github.com/mailru/go-clickhouse/v2 v2.3.0 h1:saJVHQEYZSSb0JqUc2x9l6CyT/ioYsj3qdbx8s+V0Vc=
+github.com/mailru/go-clickhouse/v2 v2.3.0/go.mod h1:TwxN829KnFZ7jAka9l9EoCV+U0CBFq83SFev4oLbnNU=
```

Comment:
Us there any reason why we use `mailru/go-clickhouse` driver in `smoke-test`, while `clickhouse-go/v2` in `quesma` itself? Maybe we should unify.

Reply:
Don't remember the reason. I needed to update `mailru/go-clickhouse` to `v2` due to following problem in smoke-test https://github.com/ClickHouse/clickhouse-go/issues/380. That's something that I have not experienced in quesma itself, however I will double check that before merging this PR